Bottom: d6bc82c31d0b10a67412a69b3db1b2f4370d7f22
Top:    8f2ad4c09d3053367397eeec808cc6171a7d879d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:53:18 +0000

polygon1.c: Remove now unused memory allocation macro MemGet


---

diff --git a/src/polygon1.c b/src/polygon1.c
index d03af9c..8cccaac 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -104,10 +104,6 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#define MemGet(ptr, type) \
-  if (UNLIKELY (((ptr) = (type *)malloc(sizeof(type))) == NULL))	\
-    error(err_no_memory);
-
 #undef DEBUG_LABEL
 #undef DEBUG_ALL_LABELS
 #undef DEBUG_JUMP
